packages/desktop-client/src/components/modals/Notes.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useEffect, useState } from 'react';
+
+import { useLiveQuery } from 'loot-core/src/client/query-hooks';
+import q from 'loot-core/src/shared/query';
+
+import Check from '../../icons/v2/Check';
+import { type CommonModalProps } from '../../types/modals';
+import Button from '../common/Button';
+import Modal from '../common/Modal';
+import View from '../common/View';
+import NotesComponent from '../Notes';
+
+type NotesProps = {
+  modalProps: CommonModalProps;
+  id: string;
+  name: string;
+  onSave: (id: string, notes: string) => void;
+};
+
+export default function Notes({ modalProps, id, name, onSave }: NotesProps) {
+  const data = useLiveQuery(() => q('notes').filter({ id }).select('*'), [id]);
+  const originalNotes = data && data.length > 0 ? data[0].note : null;
+
+  const [notes, setNotes] = useState(originalNotes);
+  useEffect(() => setNotes(originalNotes), [originalNotes]);
+
+  function _onClose() {
+    modalProps?.onClose();
+  }
+
+  function _onSave() {
+    if (notes !== originalNotes) {
+      onSave?.(id, notes);
+    }
+
+    _onClose();
+  }
+
+  return (
+    <Modal
+      title={`Notes: ${name}`}
+      showHeader={true}
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
      showHeader
```